<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notetemplate</class>
 <widget class="QWidget" name="notetemplate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>500</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string/>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_3"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_4"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>41</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_2">
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>0</height>
     </size>
    </property>
    <attribute name="title">
     <string/>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_5"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_6"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_7"/>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="pushButton_8">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_8"/>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>41</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string/>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QPushButton" name="pushButton_9">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_9"/>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QPushButton" name="pushButton_10">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_10"/>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QPushButton" name="pushButton_11">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_11"/>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string/>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QPushButton" name="pushButton_12">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_12"/>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QPushButton" name="pushButton_13">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_13"/>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QPushButton" name="pushButton_14">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_14"/>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QPushButton" name="pushButton_15">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_15"/>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>41</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>510</y>
     <width>98</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="quitBtn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>510</y>
     <width>98</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
